iOS Example Frame/iOS Example Frame/Factory/Factory+UITest.swift

```diff
@@ -74,5 +90,66 @@ extension ThemeDemo {
                             billingFormStyle: billingFormStyle)
     }
 
+    static func buildCompleteUITest() -> PaymentStyle {
```

Comment:
Is there UI test for current default screen without duplicating the code?

Reply:
If we were to do UI tests without specific UI customisations designed for the UI tests we would end up with one of the following:
- demo designs that are not what UX wants
- UI tests that do not test for "edgy" behaviours

This can be improved by creating our demo designs to match UX templates and also contain our edge case scenarios, but then it may extend the number of buttons, making it harder to digest on first sight and would make the UI tests flakier as their UI itself may become subject to change on UX whim. 
Using the specific scheme with unique template overrides ensures the UI test is separated from the demo UI we present to clients.

Reply:
I think, one of ACs to test default screen as a normal user. 
[- _re create the same UI test with new UI flow_](https://checkout.atlassian.net/browse/PIMOB-1360)

Reply:
Could you please justify why you think so? And what does "default screen as a normal user" actually mean to you, because I am unsure we have the same understanding for that.